packages/commonwealth/test/devnet/integration/integrationUtils/process-setup/setupRabbitMq.ts

```diff
@@ -0,0 +1,20 @@
+import { GenericContainer, Wait } from 'testcontainers';
+import { config } from '../../../../../server/config';
+
+export async function setupRabbitMq() {
```

Comment:
This looks like another RMQ adapter for test containers, you can continue using ports if you use this adapter

Reply:
This is not the adapter but the init function to start up the rmq in a separate docker container via the testcontainers library